python/shotgun_desktop/upgrade_startup.py

```diff
@@ -74,14 +76,10 @@ def upgrade_startup(splash, sgtk, app_bootstrap):
 
     try:
         latest_descriptor = current_desc.find_latest_version()
-    # This happens when a proxy blocks the connection (derives from ProxyError)
-    except httplib2.socks.HTTPError:
-        logger.warning("A proxy blocked the access to tank.shotgunstudio.com.")
-        return False
     # This happens when someone runs a local server and the Desktop doesn't have
     # internet access for security reasons.
-    except httplib2.ServerNotFoundError:
-        logger.warning("Could not find tank.shotgunstudio.com. Internet connection down?")
+    except (httplib2.HttpLib2Error, httplib2.socks.HTTPError, httplib.HTTPException), e:
+        logger.warning("Could not access tank.shotgunstudio.com (%s)." % e.__class__.__name__)
```

Comment:
For what it's worth, Manne suggested a while back that we use the % notation when logging since when we do the new logging in Toolkit we'll only have to update the method calls and not redo the args. In theory. :)